Tests/SentryTests/SentryBinaryImageCache+Private.h

```diff
@@ -0,0 +1,13 @@
+#import "SentryBinaryImageCache.h"
+#import "SentryCrashBinaryImageCache.h"
+
+@interface
+SentryBinaryImageCache ()
+
+@property (nonatomic, strong) NSArray<SentryBinaryImageInfo *> *cache;
```

Comment:
`l`: If the class is internal, I think we don't have to add a personal category to make it testable. IMO, you could also just add these methods directly to the class itself and add a comment that they are only required for testing, but we can also keep it if you prefer.